deps/npm/bin/npx

```diff
@@ -1,35 +0,0 @@
-#!/bin/sh
```

Comment:
If this file was intentionally removed then the Windows packaging and installer scripts will need to be updated.